data_structures/stacks/min_stack.py

```diff
@@ -1,11 +1,19 @@
 class MinStack:
+    """ Designing a Stack that can return Minimum element in Constant time"""
+
     def __init__(self):
         """ Creating stack minStack and size Variable"""
         self.stack = []
         self.minStack = []
         self.size = 0
```

Comment:
Please see https://github.com/TheAlgorithms/Python/pull/2477#discussion_r494864246 and implement [`.__iter__()`](https://docs.python.org/3/reference/datamodel.html#object.__iter__) and `.__len__()` methods for this class.  `.__iter__()` should return values in min-to-max order.